anvill/include/anvill/Lifters/Options.h

```diff
@@ -106,6 +110,9 @@ class LifterOptions {
   // Target module into which code will be lifted.
   llvm::Module *const module;
 
+  // The control flow provider, used for thunk redirections
```

Comment:
More elaborate comment needed, and also use periods in your sentences. The control-flow provider is used for thunk redirection (which helps to produce nicer bitcode for external function calls). It also provides possible targets for indirect jumps that are represented by jump tables.